pkg/virt-controller/watch/vm.go

```diff
@@ -217,6 +220,30 @@ func (c *VMController) execute(key string) error {
 		}
 	}
 
+	for _, volume := range vm.Spec.Template.Spec.Volumes {
+		if volume.VolumeSource.PersistentVolumeClaim != nil {
+			key := fmt.Sprintf("%s/%s", vm.Namespace, volume.VolumeSource.PersistentVolumeClaim.ClaimName)
+			obj, exists, err := c.pvcInformer.GetStore().GetByKey(key)
+			if exists && err == nil {
```

Comment:
Is it safe to silently ignore the error here?

Reply:
technically, a pvc that doesn't exist will get caught as an error when creating the pod, but I think it's a good practice to handle the error here as well.